site/src/pages/docs/server-annotated-service.mdx

```diff
@@ -753,6 +753,80 @@ public class MyAnnotatedService {
 }
 ```
 
+## Specifying a blocking task executor
+
+An annotated service is executed by an [EventLoop](https://netty.io/4.1/api/io/netty/channel/EventLoop.html),
+so you must not make a blocking call within the `EventLoop`. If you want to make a blocking call,
+you should delegate the call to another thread. You can use your own thread or a blocking task executor that
+Armeria provides.
+
+If you specify the <type://Blocking> annotation to the method, the method will be executed by
+the blocking task executor that is returned by <type://ServiceRequestContext#blockingTaskExecutor()>:
+
+```java
+public class MyAnnotatedService {
+
+    @Blocking // 👈
+    @Get("/hello")
+    public HttpResponse hello(ServiceRequestContext ctx) {
+        // ctx.blockingTaskExecutor() executes this method.
+        assert !ctx.eventLoop().inEventLoop();
+
+        Statement stmt = ...;
+        // Make a blocking call.
+        ResultSet resultSet = stmt.executeQuery(...);
+        HttpResponse res = responseFrom(resultSet);
+        return res;
+    }
+```
+
+<Tip>
+
+You can apply the <type://Blocking> annotation to the class level so all methods are executed by the
+blocking task executor:
+
+```java
+@Blocking // 👈
+public class MyBlockingAnnotatedService {
+    @Get("/hello")
+    public HttpResponse hello(ServiceRequestContext ctx) {
+        // ctx.blockingTaskExecutor() executes this method.
+        ...
+    }
+    @Get("/hello2")
+    public HttpResponse hello2(ServiceRequestContext ctx) {
+        // ctx.blockingTaskExecutor() executes this method.
+        ...
+    }
+}
+```
+
+</Tip>
+
+Or, you can directly use the executor returned by <type://ServiceRequestContext#blockingTaskExecutor()>
+in the method:
```

Comment:
How about this:

    You can also submit a task to the <type://ServiceRequestContext#blockingTaskExecutor()> or
    any `Executor` instead of using the `<type://@Blocking>` annotation.

Reply:
Thanks all fixed. 😉